fedimint-testing/src/gateway.rs

```diff
@@ -114,6 +114,7 @@ impl GatewayTest {
             listen,
             address.clone(),
             cli_password.clone(),
+            None, // Use default Network
```

Comment:
Nit: maybe add "(regtest)" to the comment so the reader doesn't need to go lookup what the default network is